docs/changelog/111770.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 111770
+summary: Integrate IBM watsonx to Inference API for text embeddings
+area: Experiences
+type: enhancement
+issues: []
```

Comment:
@maxhniebergall, @prwhelan What should we put in the issues field?

Reply:
you can leave it as it is.

Reply:
The issues filed is used if this pull request closes an existing open issue. Most of the time these are bug fixes. In this case you can leave it empty.